### PR TITLE
Feat/list current members

### PR DESCRIPTION
**Pull request by swellander**
## Description

Adds a new page listing all builders and linking to their profile pages.

https://github.com/BuidlGuidl/batch1.buidlguidl.com/assets/22231097/3acf34ab-13b3-4efc-99ec-e186b34d1070

## Additional Information
- Also updates the [`<Address/>` component](https://github.com/BuidlGuidl/batch1.buidlguidl.com/blob/main/packages/nextjs/components/scaffold-eth/Address.tsx) to accept an optional `hideCopyIcon` prop

## Related Issues
#3 

Your ENS/address: `websurfer.eth`

### CODE REVIEW

**Comment by dtnwen** [comment]
As the [contribute guide](https://github.com/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md#getting-started) mentions, I believe you should first raise an issue for a feature request and only make a PR after it has been accepted as a issue

> Issues should be used to report problems, request a new feature, or discuss potential changes before a PR is created.

File: packages/nextjs/pages/builders/index.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+const BuilderPage: NextPage = () => {
+  const {
+    data: eventData,
+    isLoading,
+    error,
+  } = useScaffoldEventHistory({
+    contractName: "BatchRegistry",
+    eventName: "CheckedIn",
+    fromBlock: 0n,
```
Comment (carletex) [inline packages/nextjs/pages/builders/index.tsx:14]: Let's use the block that the contract was deployed on OP, so this is faster/efficient

**Review by carletex — CHANGES_REQUESTED** [review]
This looks great @swellander 

Love the skeleton :D 

Left a tiny comment, but this is almost ready to go!

File: packages/nextjs/pages/builders/index.tsx (hunk 1 above)
Comment (swellander) [reply packages/nextjs/pages/builders/index.tsx:14]: cool, good call :ok_hand:

**Comment by swellander** [comment]
@carletex thanks again for the review :pray: pushed up the change you suggested

**Review by carletex — APPROVED** [review]
Good stuff, thanks @swellander !!

**Pull request merged**